content/en/docs/ops/integrations/spire/index.md

```diff
@@ -137,50 +164,204 @@ Istio will become the Envoy SDS listener if the socket is not created by SPIRE b
+    apiVersion: install.istio.io/v1alpha1
+    kind: IstioOperator
+    metadata:
+      namespace: istio-system
+    spec:
+      profile: default
+      meshConfig:
+        trustDomain: example.org
+      values:
+        global:
+        # This is used to customize the sidecar template
+        sidecarInjectorWebhook:
+          templates:
+            spire: |
+              spec:
+                containers:
+                - name: istio-proxy
+                  volumeMounts:
+                  - name: workload-socket
+                    mountPath: /run/secrets/workload-spiffe-uds
+                    readOnly: true
+                volumes:
+                  - name: workload-socket
+                    csi:
+                      driver: "csi.spiffe.io"
+                      readOnly: true
+      components:
+        ingressGateways:
+          - name: istio-ingressgateway
+            enabled: true
+            label:
+              istio: ingressgateway
+            k8s:
+              overlays:
+                - apiVersion: apps/v1
+                  kind: Deployment
+                  name: istio-ingressgateway
+                  patches:
+                    - path: spec.template.spec.volumes.[name:workload-socket]
+                      value:
+                        name: workload-socket
+                        csi:
+                          driver: "csi.spiffe.io"
+                          readOnly: true
+                    - path: spec.template.spec.containers.[name:istio-proxy].volumeMounts.[name:workload-socket]
+                      value:
+                        name: workload-socket
+                        mountPath: "/run/secrets/workload-spiffe-uds"
+                        readOnly: true
+                    - path: spec.template.spec.initContainers
+                      value:
+                        - name: wait-for-spire-socket
+                          image: busybox:1.28
+                          volumeMounts:
+                            - name: workload-socket
+                              mountPath: /run/secrets/workload-spiffe-uds
+                              readOnly: true
+                          env:
+                            - name: CHECK_FILE
+                              value: /run/secrets/workload-spiffe-uds/socket
+                          command:
+                            - sh
+                            - "-c"
+                            - |-
+                              echo "$(date -Iseconds)" Waiting for: ${CHECK_FILE}
+                              while [[ ! -e ${CHECK_FILE} ]] ; do
+                                echo "$(date -Iseconds)" File does not exist: ${CHECK_FILE}
+                                sleep 15
+                              done
+                              ls -l ${CHECK_FILE}
+    EOF
+    {{< /text >}}
+
+1. Apply the configuration:
+
+    {{< text syntax=bash snip_id=none >}}
+    $ istioctl install --skip-confirmation -f ./istio.yaml
```

Comment:
With the changes to the demo's workflow, this call should no longer hang.

Reply:
I was thinking of adding tests and updating the demo for the manual registration option in a separate PR. Should I still include this change here? You are correct that this call will hang. Currently, this PR is focussing on the automatic registration/SPIRE controller manager integration option.

Reply:
I would, but it's up to you, esp. if you are planning to change it further in a followup PR soon.

Reply:
I want to make sure using `istioctl manifest generate` doesn't break anything unexpectedly which I can confirm with a docs test for the manual registration option (and through manual testing). For example, if I updated istioctl install to istioctl manifest generate I would need to include the cmd to create the istio-system ns since it isn't created automatically.